examples/jsm/nodes/math/MathNode.js

```diff
@@ -253,6 +253,8 @@ export default MathNode;
 
 export const EPSILON = float( 1e-6 );
 export const INFINITY = float( 1e6 );
+export const PI = float( Math.PI );
```

Comment:
Why do we need this? AFAIR, Math.PI and Math.PI2 are already cached when generating constants, so `float( Math.PI or Math.PI2 )` (or even just them without `float` -- most places in the Nodes system autoconvert values to nodes) should already be efficient. /ping @sunag @RenaudRohlinger

Reply:
I was following the logic of constants such as EPSILON and still thinking with a GLSL mind but now I recon that all these constants might not be doable with js Math. Also when thinking about a visual node editor I believe that it's where they can be relevant.